app/Resources/views/base.html.twig

```diff
@@ -105,16 +105,7 @@
 
                     <div id="sidebar" class="col-sm-3">
                         {% block sidebar %}
-                            <div class="section about">
-                                <div class="well well-lg">
-                                    <p>
-                                        {{ 'help.app_description'|trans|raw }}
-                                    </p>
-                                    <p>
-                                        {{ 'help.more_information'|trans|raw }}
-                                    </p>
-                                </div>
-                            </div>
+                            {{ render_esi(controller('AppBundle:Blog:sidebar')) }}
```

Comment:
@sergiu-popa, I think you can use this (instead of creating an  action):
```twig
{{ render_esi('FrameworkBundle:Template:template', { 'defaults': { 'template': 'blog/sidebar.html.twig', 'sharedAge': 600 }}) }}
```
See
http://symfony.com/doc/current/cookbook/templating/render_without_controller.html#caching-the-static-template
http://symfony.com/doc/current/reference/twig_reference.html#render-esi